### PR TITLE
Rust Foundation: July 2024 Alpha-Omega Update

### PR DESCRIPTION
**Pull request by JoelMarcey**

### CODE REVIEW

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+There has been continued progress on verifying the repos that crates say they are associated with. We are also nearly complete on moving our security to the cloud in order to have more consistent runs, analysis and data.
```
Comment (JoelMarcey) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:1]: @walterhpearce @LawnGnome If there is anything specific to allude to in the cloud infrastructure migration, please let me know. I will add a section for it in the Engineering section.

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+### Sustainability Efforts
+
+Adam continues to focus on project sustainability, focusing on repo and maintainer information. Adam gave a Python specific version of his [Quantifying Nebraska talk](https://www.youtube.com/watch?v=QMHpy_mcx0Q) at [North Bay Python](https://pretalx.northbaypython.org/nbpy-2024/talk/9EXJ7T/). He is developing tooling around this.
+
+### Development of crates.io update
+
+Tobias [published](INSERT LINK HERE) an update of the development progress of crates.io.
```
Comment (JoelMarcey) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:37]: @Turbo87 Just waiting for the link on this one.

**Comment by JoelMarcey** [comment]
cc @MarcoIeni

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md (hunk 2 above)
Comment (Turbo87) [reply alpha/engagements/2024/Rust Foundation/update-2024-07.md:37]: yeah, unfortunately I still need a review approval from a team lead to be able to merge the blog post

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Engineering
+
+### Crate repo verification progress
+
+Adam has continued working on[provenance](https://lawngno.me/blog/2024/06/10/divine-provenance.html) tracking, verifying that a given crate is actually associated with the repository it claims to be. He has now analyzed the top 5000 crates by download count and has found no obvious red flags.Verification now infers commits based on tag data if the crate file was packaged without VCS info. This finds more mismatches as a result, but they’re all innocent — build artifacts, swap files, etc. Up next is to analyze the entire crates.io corpus.
+
+### Crate deletion RFC
+
+The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is now in FCP. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
```
Comment (Turbo87) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:11]: the RFC is not in FCP yet. I've proposed FCP, but the other team members still need to take a look and agree...

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+### crates.io lib/bin detection
+
+Tobias backfilled `lib/bin` information for existing crate releases. The `has_lib` and `bin_names` columns in the crates.io database have been backfilled see [conversation](https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/lib.2Fbin.20detection)). For example, https://crates.io/crates/ripgrep finally shows `cargo install` now.
+
+### Crate specific RSS feed
+
+Thanks to Tobias, there is now a [crate-specific RSS feed](https://github.com/rust-lang/crates.io/pull/9064). You can subscribe to get all the details on the specific crates you are interested in. RSS feeds per crate are published at https://static.crates.io/rss/crates/{name}.xml. The feeds are synced with the database in a background job after every successful publish of the corresponding crate. It includes the latest 10 releases of the crate, but at least all releases within the last 24 hours.
```
Comment (Turbo87) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:19]:
```suggestion
Thanks to Tobias, there are now [crate-specific RSS feeds](https://github.com/rust-lang/crates.io/pull/9064). You can subscribe to get all the details on the specific crates you are interested in. RSS feeds per crate are published at https://static.crates.io/rss/crates/{name}.xml. The feeds are synced with the database in a background job after every successful publish of the corresponding crate. It includes the latest 10 releases of the crate, but at least all releases within the last 24 hours.
```

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md (hunk 3 above)
Comment (Turbo87) [reply alpha/engagements/2024/Rust Foundation/update-2024-07.md:11]:
```suggestion
The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is close to moving into FCP now. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
```

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+There has been continued progress on verifying the repos that crates say they are associated with. We are also nearly complete on moving our security to the cloud in order to have more consistent runs, analysis and data.
+
+## Engineering
+
+### Crate repo verification progress
+
+Adam has continued working on[provenance](https://lawngno.me/blog/2024/06/10/divine-provenance.html) tracking, verifying that a given crate is actually associated with the repository it claims to be. He has now analyzed the top 5000 crates by download count and has found no obvious red flags.Verification now infers commits based on tag data if the crate file was packaged without VCS info. This finds more mismatches as a result, but they’re all innocent — build artifacts, swap files, etc. Up next is to analyze the entire crates.io corpus.
```
Comment (MarcoIeni) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:7]:
```suggestion
Adam has continued working on [provenance](https://lawngno.me/blog/2024/06/10/divine-provenance.html) tracking, verifying that a given crate is actually associated with the repository it claims to be. He has now analyzed the top 5000 crates by download count and has found no obvious red flags. Verification now infers commits based on tag data if the crate file was packaged without VCS info. This finds more mismatches as a result, but they’re all innocent — build artifacts, swap files, etc. Up next is to analyze the entire crates.io corpus.
```

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+### Crate deletion RFC
+
+The crate deletion [RFC](https://github.com/rust-lang/rfcs/pull/3660) is now in FCP. Recall, Tobias wrote this RFC to provide crate owners with a mechanism to delete crates from crates.io under certain [conditions](https://github.com/Turbo87/rust-rfcs/blob/crates-io/crate-deletions/text/3660-crates-io-crate-deletions.md#proposal).
+
+### crates.io lib/bin detection
+
+Tobias backfilled `lib/bin` information for existing crate releases. The `has_lib` and `bin_names` columns in the crates.io database have been backfilled see [conversation](https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/lib.2Fbin.20detection)). For example, https://crates.io/crates/ripgrep finally shows `cargo install` now.
```
Comment (MarcoIeni) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:15]:
```suggestion
Tobias backfilled `lib/bin` information for existing crate releases. The `has_lib` and `bin_names` columns in the crates.io database have been backfilled (see [conversation](https://rust-lang.zulipchat.com/#narrow/stream/318791-t-crates-io/topic/lib.2Fbin.20detection)). For example, https://crates.io/crates/ripgrep finally shows `cargo install` now.
```

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md (hunk 4 above)
Comment (MarcoIeni) [inline alpha/engagements/2024/Rust Foundation/update-2024-07.md:19]:
```suggestion
Thanks to Tobias, there is now a [crate-specific RSS feed](https://github.com/rust-lang/crates.io/pull/9064). You can subscribe to get all the details on the specific crates you are interested in. RSS feeds per crate are published at https://static.crates.io/rss/crates/{name}.xml (example: https://static.crates.io/rss/crates/serde.xml). The feeds are synced with the database in a background job after every successful publish of the corresponding crate. It includes the latest 10 releases of the crate, but at least all releases within the last 24 hours.
```

File: alpha/engagements/2024/Rust Foundation/update-2024-07.md (hunk 2 above)
Comment (MarcoIeni) [reply alpha/engagements/2024/Rust Foundation/update-2024-07.md:37]: maybe we can link to the PR?

**Pull request closed without merging**